languageserver/src/main/scala/langserver/core/LanguageServer.scala

```diff
@@ -21,6 +21,9 @@ class LanguageServer(inStream: InputStream, outStream: OutputStream) extends Laz
         completionRequest(textDocument, position)
       case ("textDocument/definition", TextDocumentDefinitionRequest(TextDocumentPositionParams(textDocument, position))) =>
         gotoDefinitionRequest(textDocument, position)
+      case ("textDocument/hover", TextDocumentDefinitionRequest(TextDocumentPositionParams(textDocument, position))) =>
```

Comment:
Usually I need to add this in [Commands.scala](https://github.com/dragos/dragos-vscode-scala/blob/master/languageserver/src/main/scala/langserver/messages/Commands.scala#L191), I'm surprised this is enough. If you add it there, the json-rpc library requires that types don't overlap between requests (the reason why I used the fork for a while. You'd need to define a dumb class like `TextDocumentDefinitionRequest` just to carry the parameters.